cli/pcluster/config/mappings.py

```diff
@@ -165,7 +167,7 @@
             "update_policy": UpdatePolicy.IGNORED
         },
         "aws_region_name": {
-            "default": "us-east-1",  # TODO add regex
+            "default": boto3.session.Session().region_name,
```

Comment:
what happens if there is no regions configured to be used by boto3?

Reply:
None.
No exception